Add OS parameters for SLES (#1036)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-suse-operating-system.adoc

If you extract the `.iso` image to `/var/www/html/pub/installation_media/sles/15sp3/`, the path is `\http://{foreman-example-com}/pub/installation_media/sles/15sp3/`.
. Set the *Operating System Family* to `SUSE` for all SLES systems.
. Set a location and organization context for the installation media.

> . On the *Parameters* tab, add the `sle-module-basesystem-url` parameter as string with the value `\http://{foreman-example-com}/pub/installation_media/sles/15sp3/`.

Reviewer: ```suggestion
. On the *Parameters* tab, add the `sle-module-basesystem-url` parameter, select the **string** type, and enter the value `\http://{foreman-example-com}/pub/installation_media/sles/15sp3/`.
```

Note that the path depends on the extracted `.iso` image.
ifdef::orcharhino[]
+

> Add the `or_client_repo_url` parameter as string with the value `\http://{foreman-example-com}/pulp/repos/_My_Organization_/Library/custom/SLES_Client/SLES_Client_15SP3/`.

Reviewer: ```suggestion
Add the `or_client_repo_url` parameter, select the **string** type and enter the value `\http://{foreman-example-com}/pulp/repos/_My_Organization_/Library/custom/SLES_Client/SLES_Client_15SP3/`.
```
I think this is closer the the web UI if I am not mistaken?

endif::[]
+

> (SLES 12 only) Add the parameter `additional_media` as string with the value `""`.

Reviewer: Same here as comment above.
